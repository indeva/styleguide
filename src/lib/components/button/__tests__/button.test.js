import React from 'react';
import renderer from 'react-test-renderer';
import Button from '..';

describe('Button', () => {
  it('should render button properly', () => {
    const component = renderer.create(<Button>Test</Button>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
