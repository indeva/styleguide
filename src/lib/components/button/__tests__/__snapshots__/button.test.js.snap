// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render button properly 1`] = `
<button>
  Test
</button>
`;

exports[`Button should render button properly with foo 1`] = `
<button>
  Lorem
</button>
`;
